src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -456,10 +456,8 @@ export class VisualizeEmbeddable
     const input = {
       savedVis: this.vis.serialize(),
     };
-    if (this.getTitle()) {
-      input.savedVis.title = this.getTitle();
-    }
     delete input.savedVis.id;
+    _.unset(input, 'savedVis.title');
```

Comment:
When getting a Visualize embeddable by value, you can end up with duplicate titles (for example, when unlinking it from the library) - one in `explicitInput -> savedVis -> title` and the other in `explicitInput -> title`. The only title in a by value embeddable should be `explicitInput -> title`, so I unset `explicitInput -> savedVis -> title`  when grabbing the embeddable by value.